CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ if(DEFINED ENV{QUDA_BUILD_TYPE})
 else()
   set(DEFBUILD "DEVEL")
 endif()
+if(DEFINED ENV{QUDA_BACKEND})
```

Comment:
Might as well put this comment in the first file: I do not like the language backend.  We should be talking about different targets, e.g., CUDA, OpenMP, pSTL are all compilation targets.  Apologies for the implied global edit.